recipes/raccess/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "raccess" %}
+{% set version = "1.0.0" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+   url: https://github.com/gterai/raccess/archive/ad29d098b62606aa4051951469780b4e7be54536.tar.gz
+   sha256: 70bc9b26d74c6535395c6ea75bcd5f6fdb4fadab2ad3137f66f01397eae1965c
+   patches:
+    - raccess_fix.patch
+    - raccess_makefile.patch
+    - raccess_bugfix.patch
+    - raccess_fix_link.patch
+build:
+   number: 0
+   skip: true # [osx]
+   run_exports:
+     - {{ pin_subpackage('raccess', max_pin='x.x') }}
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - zlib
+  run: []
+
+test:
+  commands:
+    - test -x "${PREFIX}/bin/raccess"
+
+about:
+  home: https://github.com/gterai/raccess
+  license: MIT
```

Comment:
Is this really MIT, [COPYRIGHT_LICENSE_DISCLAIMER.txt](https://github.com/gterai/raccess/blob/master/COPYRIGHT_LICENSE_DISCLAIMER.txt)